README.rst

```diff
@@ -10,19 +10,21 @@ pyfar
 .. image:: https://circleci.com/gh/pyfar/pyfar.svg?style=shield
     :target: https://circleci.com/gh/pyfar/pyfar
 .. image:: https://mybinder.org/badge_logo.svg
-    :target: https://mybinder.org/v2/gh/pyfar/gallery/main?filepath=docs/gallery
+    :target: https://mybinder.org/v2/gh/pyfar/gallery/main?labpath=docs/gallery/interactive/pyfar_introduction.ipynb
 
 
 The python package for acoustics research (pyfar) offers classes to store
 audio data, filters, coordinates, and orientations. It also contains common
-functions for digital audio signal processing.
+functions for digital audio signal processing and plotting audio signals.
 
 Getting Started
 ===============
 
-Check out `pyfar.org`_ for a tour through the pyfar
-universe, including complete documentation of this package and
-the other packages.
+The `pyfar workshop`_ gives an overview of the most important pyfar
+functionality and is a good starting point. It is part of the
+`pyfar example gallery`_ that also contains more specific and in-depth
+examples. The  `pyfar documentation`_ gives a detailed and complete
+overview of pyfar. All these information are available from `pyfar.org`_.
```

Comment:
```suggestion
For a list of application examples including introductions to most important functionality check out the `pyfar example gallery`_ and `pyfar workshop`_ . All gallery example notebooks can be executed interactively without a local installation by clicking the mybinder.org button on the respective example.
A full overview of all features is found in the `pyfar documentation`_ .
```

Reply:
I actually prefer the current version. It first links the workshop, which might be the easiest entry to the entrire package and explicitly links the gallery, not only pyfar.org. Would you be fine with leaving it as it is?